modules/store/src/store.ts

```diff
@@ -62,15 +62,28 @@ export class Store<T> extends Observable<T> implements Observer<Action> {
     key5: e,
     key6: f
   ): Observable<T[a][b][c][d][e][f]>;
-  /**
-   * This overload is used to support spread operator with
-   * fixed length tuples type in typescript 2.7
-   */
-  select<K = any>(...paths: string[]): Observable<K>;
+  select<
```

Comment:
I added this signature to allow invocation with an arbitrary number of path tokens, while avoiding invocations with less than 6 path tokens to fall back to the `select<K = any>(...paths: string[]): Observable<K>` signature if no valid key is passed.